crypto/srp/srp_lib.c

```diff
@@ -11,6 +11,12 @@
  * for the EdelKey project.
  */
 
+/*
+ * SRP low level APIs are deprecated for public use, but still ok for
+ * internal use.
+ */
+#include "internal/deprecated.h"
```

Comment:
`#define OPENSSL_SUPPRESS_DEPRECATED` should be sufficient as this is all in OPENSSL_NO_SRP.

Reply:
Fixed.